Doc/whatsnew/3.9.rst

```diff
@@ -109,6 +109,14 @@ New Modules
 Improved Modules
 ================
 
+datetime
+--------
+The :meth:`~datetime.date.isocalendar()` of :class:`datetime.date`
```

Comment:
This wording seems overly verbose. I suggest:

"The ``isocalendar()`` methods of :class:`datetime.date` and :class:`datetime.datetime` now return ..."